ti/instrumentation/python/helper.go

```diff
@@ -43,11 +45,10 @@ func getPythonTestsFromPattern(tests []ti.RunnableTest, workspace, testpattern s
 // In case of errors, return empty list
 func GetPythonTests(workspace string, testGlobs []string) []ti.RunnableTest {
 	tests := make([]ti.RunnableTest, 0)
-	tests, err := getPythonTestsFromPattern(tests, workspace, "test_*.py")
-	if err != nil {
-		return tests
+	if len(testGlobs) == 0 {
+		testGlobs = append(testGlobs, "test_*.py", "*_test.py")
```

Comment:
Define constant as `defaultTestGlobs = []string{"test_*.py", "*_test.py"}`

Use it there
```suggestion
		testGlobs = defaultTestGlobs
```